lang/scala/scala.talon

```diff
@@ -0,0 +1,42 @@
+mode: user.scala
```

Comment:
I recently found out there's a bit more of an incantation needed here in the matcher to avoid binding commands in sleep mode. The reason is once you add an explicit mode, the implicit command mode filter is removed.

```
mode: command
and mode: user.scala
mode: command
and mode: user.auto_lang
and code.language: scala
```